scripts/foundry/governance/RevokeMultiSig.s.sol

```diff
@@ -0,0 +1,266 @@
+                bytes memory internalTx = abi.encodePacked(isDelegateCall, to, value, dataLength, data);
+                transactions = abi.encodePacked(transactions, internalTx);
+            }
+
+            /** Remove the multisig as guardian of Angle Distributor  */
+            {
+                to = _chainToContract(chainId, ContractType.AngleDistributor);
+                bytes memory data = abi.encodeWithSelector(
+                    IAccessControl.revokeRole.selector,
+                    guardianRole,
+                    address(_chainToContract(chainId, ContractType.GovernorMultisig))
+                );
+                uint256 dataLength = data.length;
+                bytes memory internalTx = abi.encodePacked(isDelegateCall, to, value, dataLength, data);
+                transactions = abi.encodePacked(transactions, internalTx);
+            }
+
+            /** Remove the multisig as governor of Angle Distributor  */
+            {
+                to = _chainToContract(chainId, ContractType.AngleDistributor);
+                bytes memory data = abi.encodeWithSelector(
+                    IAccessControl.revokeRole.selector,
+                    governorRole,
+                    address(_chainToContract(chainId, ContractType.GovernorMultisig))
+                );
+                uint256 dataLength = data.length;
+                bytes memory internalTx = abi.encodePacked(isDelegateCall, to, value, dataLength, data);
+                transactions = abi.encodePacked(transactions, internalTx);
+            }
+
+            /** Add the on chain governor as governor of Angle Middleman */
+            {
+                to = _chainToContract(chainId, ContractType.AngleMiddleman);
+                bytes memory data = abi.encodeWithSelector(
+                    IAccessControl.grantRole.selector,
+                    guardianRole,
+                    address(_chainToContract(chainId, ContractType.Governor))
+                );
+                uint256 dataLength = data.length;
+                bytes memory internalTx = abi.encodePacked(isDelegateCall, to, value, dataLength, data);
+                transactions = abi.encodePacked(transactions, internalTx);
+            }
+
+            /** Remove the multisig as guardian of Angle Distributor  */
+            {
+                to = _chainToContract(chainId, ContractType.AngleMiddleman);
+                bytes memory data = abi.encodeWithSelector(
+                    IAccessControl.revokeRole.selector,
+                    guardianRole,
+                    address(_chainToContract(chainId, ContractType.GovernorMultisig))
+                );
+                uint256 dataLength = data.length;
+                bytes memory internalTx = abi.encodePacked(isDelegateCall, to, value, dataLength, data);
+                transactions = abi.encodePacked(transactions, internalTx);
+            }
+
+            /** Change Fee Distributor admin */
```

Comment:
Maybe let's not take care of this contract in the end as we probably won't be using it ever again -> it distributed sanUSDC which is deprecated so let's disregard I guess

Reply:
Let' still comment no?